pyatlan/generator/class_generator.py

```diff
@@ -642,6 +643,7 @@ def render_module(self, asset_info: AssetInfo, enum_defs: List["EnumDefInfo"]):
                 "asset_info": asset_info,
                 "existz": os.path.exists,
                 "enum_defs": enum_defs,
+                "templates_path": TEMPLATES_DIR.absolute().as_posix(),
```

Comment:
Curious why was this change needed?

Reply:
Related to: https://github.com/atlanhq/atlan-developer-portal/pull/300

I want to run generator scripts from the project root directory, for example: `python3 pyatlan/generator/class_generator.py`. In this case, the hardcoded path in the templates is not respected. Therefore, we should use the `pathlib` path in the templates.